### PR TITLE
Docs: char function

### PR DESCRIPTION
**Pull request by godfreyd**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category (leave one):
- Documentation

### CODE REVIEW

File: docs/ru/query_language/functions/encoding_functions.md [hunk 1]
```diff
@@ -1,5 +1,68 @@
...
+char(number_1, [number_2, ..., number_n]);
+```
+
+**Параметры**
+
+- `number_1, number_2, ..., number_n` — Числовые аргументы, которые интерпретируются как целые числа. Типы: [Int](../../data_types/int_uint.md), [Float](../../data_types/float.md).
+
+**Возвращаемое значение**
+
+- Строка UTF-8 содержащая символы, полученные путем преобразования числовых аргументов.
```
Comment (alexey-milovidov) [inline docs/ru/query_language/functions/encoding_functions.md]: Absolutely unclear. This text only exposes a lack of understanding.
Needs rewrite.

File: docs/ru/query_language/functions/encoding_functions.md [hunk 2]
```diff
@@ -1,5 +1,68 @@
 # Функции кодирования
 
+## char {#char}
+
+Возвращает символьное значение данного целого значения в соответствии с таблицей ASCII. Принимает несколько числовых аргументов.
```
Comment (alexey-milovidov) [inline docs/ru/query_language/functions/encoding_functions.md]: There is no ASCII table under the hood.

File: docs/en/query_language/functions/encoding_functions.md [hunk 3]
```diff
@@ -1,7 +1,62 @@
 # Encoding functions
 
-## char
-Accepts multiple arguments of numberic types. Returns a string with the length as the number of passed arguments and each byte has the value of corresponding argument.
+## char {#char}
+
+Returns the character value of the given integer value according to the ASCII table. Accepts multiple arguments of numeric types.
```
Comment (qoega) [inline docs/en/query_language/functions/encoding_functions.md]: We need to tell what happens if number > 256 or < 0. I guess there are not only ASCII symbols supported

Comment (godfreyd) [reply docs/en/query_language/functions/encoding_functions.md]: Снимок экрана 2020-01-24 в 11 00 54 ДП
What is this answer called?

File: docs/ru/query_language/functions/encoding_functions.md (hunk 1 above)
Comment (godfreyd) [reply docs/ru/query_language/functions/encoding_functions.md]: How right?

File: docs/ru/query_language/functions/encoding_functions.md (hunk 2 above)
Comment (godfreyd) [reply docs/ru/query_language/functions/encoding_functions.md]: I'll delete the ASCII

**Comment by alexey-milovidov** [comment]
@godfreyd I've edited the text.

**Pull request merged**